projects/S2_Climate_health_analysis/convert_json_to_dicts.py

```diff
@@ -1,33 +1,39 @@
-#%%
+"""
+Script for exploring ESGF results.
+"""
+
```

Comment:
I usually try to follow PEP8 guidelines as a Python style guide:

https://www.python.org/dev/peps/pep-0008/

This includes such pedantry as grouping imports in blocks according to their library level!

1. python standard library modules (json, re)

2. third-party library modules (pandas)

3. project-level modules

Reply:
🤯